src/components/latestNews.js

```diff
@@ -1,24 +1,31 @@
 import React from "react";
-import { newsSelector} from "../selectors/newsSelector"
-import { BannerButton } from "./generics/bannerButton";
-import { DisplayBadge } from "./generics/displayBadge";
+import { newsSelector } from "../selectors/newsSelector";
+import { companySelector} from "../selectors/companySelector";
+import { statsSelector } from "../selectors/statsSelector";
+import { quoteSelector } from "../selectors/quoteSelector";
 import { DisplayWrapper } from "./generics/displayWrapper";
-import { PriceDisplay } from "./generics/priceDisplay";
 import { TitleHeader } from "./generics/titleHeader";
 import { Text } from "./generics/text";
-import { StatLine } from "./generics/statLine";
-
 import { useDispatch, useSelector } from "react-redux";
 
 export const LatestNews = () => {
   //Call necessary selectors for display data
   const dispatch = useDispatch();
-    const newsArticles = useSelector(newsSelector);
+    const newsInfo = useSelector(newsSelector);
+
+    const companyInfo = useSelector(companySelector);
+    const quoteInfo = useSelector(quoteSelector)
+    const statInfo = useSelector(statsSelector)
+    console.log(`company info ${companyInfo}`);
+    console.log(`news info ${newsInfo}`)
+    console.log(`quote info ${quoteInfo}`)
+    console.log(`stat info ${statInfo}`)
   return (
     <DisplayWrapper width="25%" mr="auto" ml="auto">
-    {newsArticles.map((article) => <div><Text variant="primary">article.headline</Text>
-        <Text variant="secondary">article.datetime</Text>
-        <Text variant="secondary">article.source</Text></div>)}
+    <TitleHeader>Latest News</TitleHeader>
+    {newsInfo && newsInfo.map((article) => <div className="newsRow"><Text variant="primary">{article.headline}</Text>
```

Comment:
Can you just fix the styling on this code (whitespace, etc) so it looks a little cleaner? Bit hard to tell what this is doing since the divisions aren't really clear.